src/main/javascript/src/store/exposures.module.js

```diff
@@ -1,38 +1,38 @@
-import { exposuresService } from '@/services'
+import { exposuresService } from "@/services";
```

Comment:
For consistency's sake, I've been avoiding semicolons and using single quotes instead of double quotes when possible. Let's try to stay consistent with that approach.

Reply:
Updating